terraform/modules/jenkins/s3_bucket.tf

```diff
@@ -0,0 +1,28 @@
+resource "aws_s3_bucket" "jenkins_logs_bucket" {
+  bucket        = "${var.name}-${var.dns_name}-logs-bucket"
+  force_destroy = true
+
+  policy = <<POLICY
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "AWSElasticLoadBalancerWrite",
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": [
+                    "arn:aws:iam::156460612806:root"
```

Comment:
Which user is this? Should we be hardcoding this ID?

Reply:
Ah, good spot.

This is an annoying thing about the Elastic Load Balancer, the way to include it in a policy is just to include this account ID from [this table](https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/enable-access-logs.html#w281aac21b9c17c11b9b8b3b5b3) in the AWS docs.

It would be nice to document this somewhere, but I don't think you can put comments in JSON...

Reply:
Comment added to just above the policy definition